library/core/src/ptr/const_ptr.rs

```diff
@@ -132,8 +132,8 @@ impl<T: ?Sized> *const T {
     /// ```
     #[unstable(feature = "ptr_to_from_bits", issue = "91126")]
     #[deprecated(
-        since = "1.67",
-        note = "replaced by the `exposed_addr` method, or update your code \
+        since = "1.67.0",
```

Comment:
Hmm, was this not set to `CURRENT_RUSTC_VERSION` and automatically changed on the release 🤔

Reply:
No (#95583)

Reply:
Oh, oops 🙃 

TBH, these methods should just be deleted now.  The new strict-provenance-related ones are the way forward.